exercises/acronym/example.rs

```diff
@@ -0,0 +1,27 @@
+///
+/// Techies love their TLA (Three Letter Acronyms)!
+///
+/// Help generate some jargon by writing a program that converts a long name
+/// like Portable Network Graphics to its acronym (PNG).
+#[allow(unused_variables)]
```

Comment:
this can be removed from the example, right? The example probably does not have any remaining unused_variables.